content/remote-development/getting-started.md

```diff
@@ -28,6 +28,13 @@ If the GitHub repository is not the origin of your checked out project then you
 
 ### OSX (64-bit):
 
+If you use [Homebrew](https://brew.sh/), you can install cp-remote via:
```

Comment:
Is that a personal preference or a wider standard? (genuine question) The difficulty is that I'm trying to maintain consistency across the document and the rest of the documents. Rightly or wrongly, backticks have been used for much more than just command expressions - really they are being used to highlight anything that looks like a command, application name, variable or filename. This is more than I would have done if I'd written everything from scratch, but presentationally it is useful as it distinguishes technical terms from regular content.